DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/_CentreCourseCardAdminFields.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup
+@model SearchableCourseStatisticsViewModel
+
+<b>Admin fields and response summary</b>
+
+@foreach (var prompt in Model.AdminFieldWithResponseCounts) {
+  <dl class="nhsuk-summary-list">
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key nhsuk-u-font-weight-normal">
+        @prompt.CustomPromptText
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        <dl class="nhsuk-summary-list">
+          @foreach (var responseCount in prompt.ResponseCounts) {
+            <div class="nhsuk-summary-list__row basic-summary-list__row">
+              <dt class="nhsuk-summary-list__key nhsuk-u-font-weight-normal">
+                @responseCount.Response
+              </dt>
+              <dd class="nhsuk-summary-list__value admin-field-count">
+                @responseCount.Count
+              </dd>
+            </div>
+          }

```

Comment:
I think it's fine, but I have no idea if it would be confusing. We need more control with a table like structure than the examples on the NHS site both of which could cause a word wrapped mess with differing answer lengths.

Reply:
Good point, this is probably fine as is then.